src/framework/theme/components/card/flip-card/flip-card.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="flipcard-container">
+  <div class="flipcard nb-card-{{size}}" [ngClass]="{'flipped': flipped}">
+    <div class="card-front-container">
+      <ng-content select=".card-front"></ng-content>
```

Comment:
Would be better to introduce `flip-card-front` `flip-card-back` directive to use as selectors. So that we won't use classes for non-css purposes.